### PR TITLE
fix: only set CA data when necessary

### PR DESCRIPTION
**Pull request by jr0d**
**What type of PR is this?**

Bug

**What this PR does/ why we need it**:

windows instructions no longer contain certificate-authority option when using the system certificate store

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/D2IQ-70889

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
[dex-k8s-authenticator] fixed bug causing windows instructions to be invalid when using system certificates.
```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [x] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

File: staging/dex-k8s-authenticator/html-templates/windows-tab.html [hunk 1]
```diff
@@ -85,8 +85,14 @@ <h3>Run configuration commands</h3>
     <button class="btn" style="float:right" data-clipboard-snippet="">
       <img class="clippy" width="13" src="{{ .Web_Path_Prefix }}static/clippy.svg" alt="">
     </button>
+
+    {{ if .K8sCaPem }}
```
Comment (shaneutt) [inline staging/dex-k8s-authenticator/html-templates/windows-tab.html:89]: is the whitespace ahead of the logic block right here? I take it it works either way? :thinking:

**Comment by jr0d** [comment]
test is failing waiting on dummylb to come up
```
+ install_dummylb
+ echo 'Installing dummylb...'
+ DUMMYLB_SHA=cb4c17d70e63393f8de7cfa97d186aa06e781b3cd25bfff1f374b9d57159e80f
+ DUMMYLB_REG=registry.gitlab.com/joejulian/dummylb
Installing dummylb...
+ curl -sL https://gitlab.com/joejulian/dummylb/-/raw/f5c51f24e706cd4c5ebe7e5d36e688d167473f8b/dummylb.yaml
+ sed 's%image: registry.gitlab.com/joejulian/dummylb:latest%image: registry.gitlab.com/joejulian/dummylb@sha256:cb4c17d70e63393f8de7cfa97d186aa06e781b3cd25bfff1f374b9d57159e80f%'
+ docker_exec kubectl apply -f -
+ docker exec --interactive ct kubectl apply -f -
clusterrole.rbac.authorization.k8s.io/dummylb created
serviceaccount/dummylb created
clusterrolebinding.rbac.authorization.k8s.io/dummylb created
deployment.apps/dummylb created
+ docker_exec kubectl wait --for=condition=Available --selector=app=dummylb deploy
+ docker exec --interactive ct kubectl wait --for=condition=Available --selector=app=dummylb deploy
error: timed out waiting for the condition on deployments/dummylb
```

File: staging/dex-k8s-authenticator/html-templates/windows-tab.html (hunk 1 above)
Comment (jr0d) [reply staging/dex-k8s-authenticator/html-templates/windows-tab.html:89]: I added it for readability. It doesn't make any difference in the rendering.

**Comment by alejandroEsc** [comment]
> test is failing waiting on dummylb to come up
> 
> ```
> + install_dummylb
> + echo 'Installing dummylb...'
> + DUMMYLB_SHA=cb4c17d70e63393f8de7cfa97d186aa06e781b3cd25bfff1f374b9d57159e80f
> + DUMMYLB_REG=registry.gitlab.com/joejulian/dummylb
> Installing dummylb...
> + curl -sL https://gitlab.com/joejulian/dummylb/-/raw/f5c51f24e706cd4c5ebe7e5d36e688d167473f8b/dummylb.yaml
> + sed 's%image: registry.gitlab.com/joejulian/dummylb:latest%image: registry.gitlab.com/joejulian/dummylb@sha256:cb4c17d70e63393f8de7cfa97d186aa06e781b3cd25bfff1f374b9d57159e80f%'
> + docker_exec kubectl apply -f -
> + docker exec --interactive ct kubectl apply -f -
> clusterrole.rbac.authorization.k8s.io/dummylb created
> serviceaccount/dummylb created
> clusterrolebinding.rbac.authorization.k8s.io/dummylb created
> deployment.apps/dummylb created
> + docker_exec kubectl wait --for=condition=Available --selector=app=dummylb deploy
> + docker exec --interactive ct kubectl wait --for=condition=Available --selector=app=dummylb deploy
> error: timed out waiting for the condition on deployments/dummylb
> ```

This should be fixed now, bumped to latest.

**Pull request merged**